### DISCUSSION TITLE
Axios breaks on updating

### DISCUSSION THREAD

**Discussion by anubhavRaj27** [axios/axios#6600] [Q&A]
Hey I am was using axios version 0.21.0 but when i updated it the app stopped working , i can see the request go in the browser and the response also coming back but for some reason axios is not able to recognise or get the response , this only happens when i upgrade from version 0.21.0 to higher versions it works for 0.21.0

**Comment by ujamX** [comment]
Hi, @anubhavRaj27 

The difference between v0.21.0 and the current version (v1.7.8) is very large.
Due to feature additions and changes, code that worked well in previous versions may not work in newer versions.
Problems encountered while upgrading to the latest version are most likely due to feature changes or compatibility issues.
It is recommended that you refer to the changelog to identify differences and gradually modify the code.
Best of luck 🍀
